common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -109,16 +116,17 @@ public String getTranslatedName() {
 
     @Override
     public String getTranslation(String keySuffix) {
-        return I18n.get("feature.wynntils" + getDeclaringFeatureNameCamelCase() + ".overlay." + getNameCamelCase() + "."
-                + keySuffix);
+        return I18n.get("feature.wynntils." + getDeclaringFeatureNameCamelCase() + ".overlay." + getNameCamelCase()
```

Comment:
How come we did not notice this bug? 🤔 All those bug fixes we see scare me a bit. We have no automatic testing, and apparently even our ad-hoc testing is not good enough...

Reply:
To be fair this was a written when translations where not used anywhere for overlays so we couldn't see wrong behavior even with testing.